test/e2e/subscription_e2e_test.go

```diff
@@ -2813,48 +2816,58 @@ func checkDeploymentWithPodConfiguration(t GinkgoTInterface, client operatorclie
 		return
 	}
 
-	check := func(container *corev1.Container) {
+	check := func(container *corev1.Container) error {
```

Comment:
:+1: I've had the similar urge to change this signature when tackling other e2e work.